### PR TITLE
[OrderByComparator] LPD 27020

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPD-27020](https://liferay.atlassian.net/browse/LPD-27020)
- **LPD-27020 Add singleton to asset-list-api**
- **LPD-27020 Apply asset-list-api usages**
- **LPD-27020 Apply asset-list-api test usage**
- **LPD-27020 Baseline**
- **LPD-27020 Add singleton to AppTitleComparator**
- **LPD-27020 Apply AppTitleComparator usage**
- **LPD-27020 Baseline**
- **LPD-27020 Add Upgrade case and test**

[LPD-27020]: https://liferay.atlassian.net/browse/LPD-27020?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:report:74760972

**Comment by victorhcunha** [comment]
Pushing only to fix breaking change commit message

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
Waiting for Testray import to analyze the tests results.

**Comment by victorhcunha** [comment]
ci:report:74848107

**Comment by victorhcunha** [comment]
ci:report:74834991

**Comment by victorhcunha** [comment]
ci:test:relevant

**Comment by victorhcunha** [comment]
ci:test:upgrade

**Comment by victorhcunha** [comment]
ci:report:75128256

**Comment by victorhcunha** [comment]
ci:report:75113596

**Comment by victorhcunha** [comment]
ci:report:75113596

File: modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/util/comparator/AssetListEntryUsageModifiedDateComparator.java [hunk 1]
```diff
@@ -23,7 +23,9 @@ public class AssetListEntryUsageModifiedDateComparator
 
 	public static final String[] ORDER_BY_FIELDS = {"modifiedDate"};
 
-	public static AssetListEntryUsageModifiedDateComparator getInstance(boolean ascending) {
+	public static AssetListEntryUsageModifiedDateComparator getInstance(
```
Comment (dantewang) [inline modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/util/comparator/AssetListEntryUsageModifiedDateComparator.java:26]: @victorhcunha This change should be in the previous commit, and be logged in the replacement?

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/util/comparator/AssetListEntryUsageModifiedDateComparator.java (hunk 1 above)
Comment (victorhcunha) [reply modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/util/comparator/AssetListEntryUsageModifiedDateComparator.java:26]: Thanks, @dantewang !
This sf apply really should be in the previous commit.

**Comment by victorhcunha** [comment]
ci:test:sf

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/153799

**Pull request closed without merging**